src/app/pages/Houses/useHousePoints.js

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react";
+
+const FEED_URL = "https://docs.google.com/spreadsheets/d/";
+const SPREADSHEET_KEY = "1AV98P5Fs3INBtbwT69oJvfN0eAgj9daVw6o_6XIGEiI";
+const MODE = "pub";
+const SHEET_GID = "1705286477";
+
+function useHousePoints() {
+	const [housePoints, setHousePoints] = useState({});
+
+	useEffect(() => {
+		const dataURL = new URL(`${FEED_URL}${SPREADSHEET_KEY}/${MODE}`);
+		dataURL.searchParams.set("gid", SHEET_GID);
+		dataURL.searchParams.set("single", "true");
+		dataURL.searchParams.set("output", "tsv");
+
+		const getHousePoints = async () => {
+			try {
+				const response = await fetch(dataURL);
+				const text = await response.text();
+
+				const data = {};
+				// Parse each tab-separated line
+				for (const line of text.split("\n")) {
+					const [key, value] = line.split("\t");
+					data[key] = value;
+				}
+
+				setHousePoints(data);
+			} catch (err) {
+				console.error("Error occurred while fetching sheets data:", err);
+			}
+		};
+
+		getHousePoints();
+	}, []);
```

Comment:
I would disagree since naming the function documents the intent and leaves open possibilities for better debugging

Reply:
if you wanted the best way to write it, you'd likely want to extract the actual fetch request into a separate function and import it in here so that you could easily write a unit test for it but im w/e with even the current iteration - not worth all the work imo

Reply:
As we say, works for now.